ReportSimulationOutput/tests/output_report_test.rb

```diff
@@ -1117,6 +1119,8 @@ def test_timeseries_local_time_dst_and_utc
     assert_equal(8760, timeseries_rows.size - 2)
     timeseries_cols = timeseries_rows.transpose
     assert_equal(3, timeseries_rows[0].select { |r| r.start_with?('Time') }.size)
+    assert_equal(1, timeseries_rows[0].select { |r| r.start_with?('TimeDST') }.size)
+    assert_equal(1, timeseries_rows[0].select { |r| r.start_with?('TimeUTC') }.size)
```

Comment:
Can we make all of these checks a bit more explicit? E.g.,
```suggestion
    assert_equal(1, timeseries_rows[0].select { |r| r == 'Time' }.size)
    assert_equal(1, timeseries_rows[0].select { |r| r == 'TimeDST' }.size)
    assert_equal(1, timeseries_rows[0].select { |r| r == 'TimeUTC' }.size)
```

(This is untested.)